Source/core/settings/ChromeSettingsProvider.js

```diff
@@ -67,11 +77,15 @@ ChromeSettingsProvider.prototype.get = function(key, def) {
  * returns CO2 emission rate
  * default emission rate provided by:
  * https://www.epa.gov/sites/production/files/2016-02/documents/420f14040a.pdf
+ * find a credible source for public transport emission in kg/hr
  * @return {number}
  */
 
-ChromeSettingsProvider.prototype.getCarbonEmission = function() {
-  return this.get('CO2emissionRate', 0.255384);
+ChromeSettingsProvider.prototype.getCarbonEmission = function(type) {
+  switch(type){
```

Comment:
I'm not sure I like this solution where "type" (distance or time) is passed as an argument and then used in a switch statement...